action.yml

```diff
@@ -7,10 +7,30 @@ inputs:
   github-token:
     description: "GitHub token to access the bob release."
     required: true
-  version:
-    description: Version of bob CLI to install
+  tag:
```

Comment:
Is this parameter used anywhere currently? If not this can be freely updated to not affect anyone. 

[As I was typing this question, I quickly glanced at https://github.com/search?q=org%3Ahashicorp+action-setup-bob+language%3AYAML&type=code&l=YAML&p=3 and I think the answer is no.]